docs/user-guide/index.md

```diff
@@ -1,15 +1,7 @@
 # Getting Started
 
-Welcome to Browsertrix Cloud. This Guide will cover various aspects of using Browsertrix Cloud.
+This section covers usage of Browsertrix Cloud.
 
+## Automated Crawling
 
-## Creating an Account
```

Comment:
I think this is good info to keep, that there are two options, invite vs registration link.

Reply:
Thoughts on moving this info to the deployment guide?  IMO that is more relevant for people running the server / setting up users than it will be as a reference to users who likely won't be looking at the manual, wondering how to sign up for the service that somebody else is provisioning for them?  Kind of on the server admin to instruct people as part of their rollout IMO, as we will do on our homepage _eventually_.

Reply:
I think as written this is exactly relevant for users - either they're given a registration link or are invited, so it makes sense to cover how users start. In the deployment guide, we can explain *how* to configure these options

Reply:
Alrighty, will update